scripts/zones/Ceizak_Battlegrounds/mobs/Unfettered_Twitherym.lua

```diff
@@ -0,0 +1,110 @@
+local entity = {}
+
+local originalDamage = 0
+local damageReductionValue = -9000
+
+local skillchainAffinities = {
+    Fire = { xi.skillchainType.FUSION, xi.skillchainType.LIQUIFACTION, xi.skillchainType.LIGHT, xi.skillchainType.LIGHT_II },
+    Water = { xi.skillchainType.REVERBERATION, xi.skillchainType.DISTORTION, xi.skillchainType.DARKNESS, xi.skillchainType.DARKNESS_II },
+    Thunder = { xi.skillchainType.IMPACTION, xi.skillchainType.FRAGMENTATION, xi.skillchainType.LIGHT, xi.skillchainType.LIGHT_II },
+    Wind = { xi.skillchainType.DETONATION, xi.skillchainType.FRAGMENTATION, xi.skillchainType.LIGHT, xi.skillchainType.LIGHT_II },
+    Ice = { xi.skillchainType.INDURATION, xi.skillchainType.DISTORTION, xi.skillchainType.DARKNESS, xi.skillchainType.DARKNESS_II },
+    Earth = { xi.skillchainType.SCISSION, xi.skillchainType.GRAVITATION, xi.skillchainType.DARKNESS, xi.skillchainType.DARKNESS_II },
+    Dark = { xi.skillchainType.COMPRESSION, xi.skillchainType.DARKNESS, xi.skillchainType.DARKNESS_II },
+    Light = { xi.skillchainType.TRANSFIXION, xi.skillchainType.LIGHT, xi.skillchainType.LIGHT_II }
+}
+
+local chosenElement = nil
+local phaseDTApplied = {
+    ['74_49'] = false,
+    ['49_24'] = false,
+    ['24_0'] = false
+}
+
+local function chooseAffinity()
+    local affinities = { 'Fire', 'Water', 'Thunder', 'Wind', 'Ice', 'Earth', 'Dark', 'Light' }
+    chosenElement = affinities[math.random(#affinities)]
```

Comment:
We have utils that can help you here:

utils.randomEntryIdx(t)

utils.randomEntry(t)